docker-compose.yml

```diff
@@ -39,6 +39,7 @@ services:
       - ATO_PUBLIC_KEY_FILE=../../shared_files/ATO_public.pem
       - ATO_PRIVATE_KEY_FILE=../../shared_files/ATO_private.pem
       - HTTP_ONLY=true
+      - ENABLE_PATIENT_EXPORT=true
```

Comment:
do we wanna leave this here?

Reply:
Without it, the endpoint isn't available for the integration tests. Talked to @msnook about it briefly yesterday, and I think we can override it by environment.

Reply:
@tbellj Yup, I think it is better to place in `docker-compose` then in `Dockerfile`.  Placing in `Dockerfile` would mean that everyone's images are now out of sync.  Much more flexible to place these vars in `docker-compose`.